coral-visualization/src/main/java/com/linkedin/coral/vis/CoraIlRVisualizationUtil.java

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2023 LinkedIn Corporation. All rights reserved.
+ * Licensed under the BSD-2 Clause license.
+ * See LICENSE in the project root for license information.
+ */
+package com.linkedin.coral.vis;
+
+import java.io.File;
+import java.io.InputStream;
+import java.util.UUID;
+
+import org.apache.calcite.sql.util.SqlVisitor;
+import org.apache.hadoop.hive.conf.HiveConf;
+import org.apache.hadoop.hive.metastore.api.MetaException;
+import org.apache.hadoop.hive.ql.metadata.Hive;
+import org.apache.hadoop.hive.ql.metadata.HiveException;
+
+import com.linkedin.coral.common.HiveMscAdapter;
+import com.linkedin.coral.hive.hive2rel.HiveToRelConverter;
+
+import guru.nidi.graphviz.attribute.Font;
+import guru.nidi.graphviz.engine.Format;
+import guru.nidi.graphviz.engine.Graphviz;
+import guru.nidi.graphviz.model.Factory;
+import guru.nidi.graphviz.model.Graph;
+import guru.nidi.graphviz.model.Node;
+
+
+public class CoraIlRVisualizationUtil {
```

Comment:
`CoralIRVisualizationUtil`?

Reply:
Removed Coral from the name of both classes (also simplified this class to operate on `SqlNode` directly).